src/components/Tabs/Sonnet.styled.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+const StyledSonnet = styled.div`
+  position: absolute;
```

Comment:
Myślę, że tutaj nie ma potrzeby korzystania z `absolute` - powinien skorzystać z bardziej elastycznego rozwiązania.
Zobacz jak to wygląda u mnie:
![image](https://user-images.githubusercontent.com/55095257/118008674-9777da00-b34d-11eb-9f5f-b3e2479ad7f5.png)

Reply:
wydaje mi sie, ze mozna to rozwiazac zwyklym display: block, ale nie jesli ten element z tekstem znajduje sie wewnatrz <Tab/>, tak jak ma to miejsce w bootsrtrapowej wersji.

Reply:
Pewnie wystarczy - dlatego zastanawiam się dlaczego postanowiłeś iść tą drogą.

Reply:
w poleceniu mamy, aby komponent Tabs zawieral state, jesli mam wykorzystac display:block oraz element z tekstem umiescic z poziomu elementu <Task04> to wlasnie ten element bedzie musial zawierac state od ktorego zalezec bedzie wyswietalenie tekstu pod wlasciwym Tabem

Reply:
Chodzi po prostu o zawartość, która ma być wyświetlona. Musi być normalny `if` który pokaże dany element lub go ukryje.